### DISCUSSION TITLE
Inline prompt frequency option terminology

### DISCUSSION THREAD

**Discussion by dkoo** [Automattic/newspack-popups#877] [Polls]
#866 removed the ability for users to manually and permanently dismiss prompts, rendering the "Until dismissed" frequency option nonsensical. #866 also renamed that option to "Every page view", but after some internal discussion we've found that verbiage to be insufficiently descriptive. What should we call this option, which essentially means that the inline prompt will be always be displayed to readers who match its segment, wherever it can be injected?

**Comment by aschweigert** [comment]
"every pageview" is the clearest option here so i think we should stick with that

**Reply by iAmSteveBeatty** [reply]
My concern with "every pageview" is that it's not accurate when you have just an inline prompt. Such prompts won't show on pages. "Always" keeps it pretty simple.

**Reply by dkoo** [reply]
Just wanted to point out that inline prompts _can_ appear on pages, not just articles, if you select "Pages" in the Post Types panel.
